src/Serilog.Sinks.Logz.Io/LogzioSink.cs

```diff
@@ -128,25 +128,41 @@ private string FormatLogEvent(LogEvent loggingEvent)
             var values = new Dictionary<string, object>
             {
                 {"@timestamp", loggingEvent.Timestamp.ToString("O")},
-                {"Level", loggingEvent.Level.ToString()},
-                {"Message", loggingEvent.RenderMessage()},
-                {"Exception", loggingEvent.Exception}
+                {"level", loggingEvent.Level.ToString()},
+                {"message", loggingEvent.RenderMessage()},
+                {"exception", loggingEvent.Exception}
             };
 
             if (loggingEvent.Properties != null)
             {
                 if (loggingEvent.Properties.TryGetValue("SourceContext", out var sourceContext))
                 {
-                    values["Logger"] = sourceContext.ToString();
+                    values["logger"] = sourceContext.ToString();
+                }
+                if (loggingEvent.Properties.TryGetValue("ThreadId", out var threadId))
+                {
+                    values["thread"] = GetPropertyInternalValue(threadId);
                 }
 
                 foreach (var property in loggingEvent.Properties)
                 {
-                    values[$"Properties.{property.Key}"] = property.Value.ToString();
+                    values[$"properties.{property.Key}"] = GetPropertyInternalValue(property.Value);
```

Comment:
Maybe it makes sense to lowercase whole key?
So we would avoid having multiple fields like properties.URL, properties.Url or properties.url.

Reply:
I second that though but it makes worry for fields with multiple words like `AccessTime`... Would it be better to consider a property name serialization strategy? Like a configurable behaviour that can convert `AccessTime` to `access_time`?

Reply:
Good point, however do not have strong opinion here. For me "properties.accesstime" is still fine :)

Reply:
I'd rather not change the names that users define in their code. (Serilog Context properties) This can lead to many problems, such as user name confusion and confusion when querying the data. And I'd rather just use the same name that the user chose.

The loj4j adapter for LogzIo uses the lowercase properties, and LogzIo needs the "message" to be lowercase to display correctly in the panel. That's because I decided to use lowercase for the main properties.

Reply:
Ok, you are right, so I am going to merge PR.